### PR TITLE
Fix SEO error about duplicated content and multiple sitemaps

### PR DESCRIPTION
**Pull request by andriyParashchuk**
Guides:

* https://support.google.com/webmasters/answer/139066?hl=en
* https://sitechecker.pro/duplicate-content/

### CODE REVIEW

**Comment by andriyParashchuk** [comment]
Staging - http://jetthoughts-pr389.surge.sh/

**Comment by andriyParashchuk** [comment]
Was added canonical url for `agentinbox.html`
@pftg @ad-sys @sweatC Please check

File: _includes/head.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
   <meta name="title" content="{{ site.title }}">
   <meta name="description" content="{{ site.description }}"/>
-  <link rel="canonical" href="{{ site.url }}" />
+  <link rel="canonical" href="{{ site.url }}{{ page.url | replace:'index.html',''}}">
```
Comment (pftg) [inline _includes/head.html:12]: please add space `replace: 'index.html'`

**Pull request merged**